packages/lesswrong/lib/routes.ts

```diff
@@ -373,155 +373,169 @@ addRoute(
   }
 );
 
-if (taggingNameIsSet.get()) {
+
+const usePluralTagName = !isLWorAF && taggingNameIsSet.get();
+const taggingNameCapitalized = usePluralTagName ? taggingNamePluralCapitalSetting.get() : taggingNameCapitalSetting.get();
+
   addRoute(
     {
-      name: 'tagsSingleCustomName',
-      path: `/${taggingNamePluralSetting.get()}/:slug`,
+      name: 'tagsSingle',
```

Comment:
I admit that I am not totally following what the refactor here (with all the routes) was and would find it easier to review if it were described in text

Reply:
Old:
if (taggingName != 'tag') -> bunch of routes that use capitalized tag name + redirects from /tag routes
else (routes that use tag)

New:
single set of routes for whatever you tag name (and optional url tag base) base
if (taggingName/url base != 'tag') include redirects